src/01-dinosaur-facts.js

```diff
@@ -22,7 +22,30 @@ const exampleDinosaurData = require("../data/dinosaurs");
  *  getLongestDinosaur(dinosaurs);
  *  //> { Brachiosaurus: 98.43 }
  */
-function getLongestDinosaur(dinosaurs) {}
+function getLongestDinosaur(dinosaurs) {
+  // Return empty object if dinosaurs is an empty array
+  if (dinosaurs.length === 0) { return {}; }
+  // A variable that will be updated to contain the longest dino object
+  let longestDino;
```

Comment:
If you initiate your `longestDino` variable with an empty object, if your dino array is empty the loop will not be executed and you can return longest dino on line 46. 

This will make it that you don't need to include the conditionals on line 27